data/pathfinder/paizo/campaign_setting/book_of_the_damned_volume_1/botd1_races.lst

```diff
@@ -4,9 +4,9 @@ SOURCELONG:Princes of Darkness, Book of the Damned, Volume 1	SOURCESHORT:BOTD1	S
 ### Entry Date: 09-2012
 
 #
-# Race Name				Unique Key				Starting Feats	Size		Move				Face/Space	Reach		Combat bonus					Stat bonus																	Modify VAR													Ability																																																														Natural Attacks										Spell Res.	Damage Reduction		Monster Class Name and Starting Level						Main Race Type	Race Subtype					Challenge Rating	Source Page		Apply Kit	Class Skill
-Devil, Apostate (Deimavigga)	KEY:Devil (Apostate)					SIZE:M	MOVE:Walk,30,Fly,30	FACE:5	REACH:10	BONUS:COMBAT|AC|12|TYPE=NaturalArmor	BONUS:STAT|STR|18	BONUS:STAT|DEX|20	BONUS:STAT|CON|18	BONUS:STAT|INT|10	BONUS:STAT|WIS|14	BONUS:STAT|CHA|18	BONUS:VAR|RegenerationRate|5	BONUS:VAR|Maneuverability|5										ABILITY:Special Ability|AUTOMATIC|Flight Maneuverability|Regeneration	ABILITY:Special Ability|AUTOMATIC|Apostate Devil ~ Boundless Reach|Apostate Devil ~ Evangelization|Apostate Devil ~ Indomitable Oration|Apostate Devil ~ Malleable Form|Apostate Devil ~ Ohrwurm|Apostate Devil ~ Spell-Like Abilities|Apostate Devil ~ Summon Devil|Apostate Devil ~ Wisdom Drain	ABILITY:Internal|AUTOMATIC|Claw													SR:27		DR:10/Good and Silver	MONSTERCLASS:Outsider (Fort/Will):18						RACETYPE:Outsider	RACESUBTYPE:Devil|Evil|Extraplanar|Lawful	CR:17			SOURCEPAGE:p.54	KIT:1|LE	CSKILL:Diplomacy|Intimidate|Knowledge (History)|Knowledge (Religion)
+# Race Name				Unique Key				Starting Feats	Size		Move				Face/Space	Reach		Combat bonus					Stat bonus																	Modify VAR													Ability																																																														Natural Attacks										Spell Res.	Damage Reduction		Monster Class Name and Starting Level						Main Race Type	Race Subtype					Challenge Rating	Source Page		Apply Kit	Class Skill		Proficiencies
+Devil, Apostate (Deimavigga)	KEY:Devil (Apostate)					SIZE:M	MOVE:Walk,30,Fly,30	FACE:5	REACH:10	BONUS:COMBAT|AC|12|TYPE=NaturalArmor	BONUS:STAT|STR|18	BONUS:STAT|DEX|20	BONUS:STAT|CON|18	BONUS:STAT|INT|10	BONUS:STAT|WIS|14	BONUS:STAT|CHA|18	BONUS:VAR|RegenerationRate|5	BONUS:VAR|Maneuverability|5										ABILITY:Special Ability|AUTOMATIC|Flight Maneuverability|Regeneration	ABILITY:Special Ability|AUTOMATIC|Apostate Devil ~ Boundless Reach|Apostate Devil ~ Evangelization|Apostate Devil ~ Indomitable Oration|Apostate Devil ~ Malleable Form|Apostate Devil ~ Ohrwurm|Apostate Devil ~ Spell-Like Abilities|Apostate Devil ~ Summon Devil|Apostate Devil ~ Wisdom Drain	ABILITY:Internal|AUTOMATIC|Claw													SR:27		DR:10/Good and Silver	MONSTERCLASS:Outsider (Fort/Will):18						RACETYPE:Outsider	RACESUBTYPE:Devil|Evil|Extraplanar|Lawful	CR:17			SOURCEPAGE:p.54	KIT:1|LE	CSKILL:Diplomacy|Intimidate|Knowledge (History)|Knowledge (Religion)						AUTO:ARMORPROF|ARMORTYPE=Heavy|ARMORTYPE=Light|ARMORTYPE=Medium	AUTO:SHIELDPROF|Tower Shield|SHIELDTYPE=Buckler|SHIELDTYPE=Heavy|SHIELDTYPE=Light
```

Comment:
Not to nitpick, but this "AUTO:ARMORPROF|ARMORTYPE=Heavy|ARMORTYPE=Light|ARMORTYPE=Medium AUTO:SHIELDPROF|Tower Shield|SHIELDTYPE=Buckler|SHIELDTYPE=Heavy|SHIELDTYPE=Light
"
Is easier to track via the troubleshooting when we use "ABILITY:Internal|AUTOMATIC|TYPE=ArmorProfHeavy|TYPE=ShieldProfTower"
Anyways, looks good otherwise.